### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by Maxr1998**

### CODE REVIEW

**Review by nvllsvm — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+RUN yarn install
+
+# Copy resources
+ADD . .
+
+# Build static site
+RUN yarn generate
+
+# Deploy to nginx
+FROM nginx:latest
```
Comment (nvllsvm) [inline Dockerfile:22]: nit - `nginx:alpine` is roughly 5x smaller

Comment (Maxr1998) [reply Dockerfile:22]: Oops yes, gonna fix. Shouldn't have copy-pasted there, oof.

**Review by MrTimscampi — APPROVED** [review]
I'm definitely not a Docker guru (I've only written a few docker-compose files :p ) but LGTM.

**Comment by Shadowghost** [comment]
Shouldn't we use `COPY` instead of `ADD` to comply with [best practice](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy)?

**Pull request merged**